src/commcare_cloud/ansible/roles/postgresql_base/defaults/main.yml

```diff
@@ -29,9 +29,13 @@ postgresql_maintenance_work_mem: '64MB'
 postgresql_checkpoint_completion_target: '0.5'
 postgresql_wal_buffers: '-1'
 postgresql_default_statistics_target: '100'
+postgresql_min_wal_size: ~
```

Comment:
what does `~` do? I always associate it with a user's home directory

Reply:
This is the canonical way to represent `null` in YAML: https://yaml.org/type/null.html

Reply:
could change to `null` if it's clearer

Reply:
if it's the canonical way I'm happy to have it. I just hadn't seen it before